docker-compose.yml

```diff
@@ -45,6 +45,7 @@ services:
       - LANTERN_CAPQUERY_QRYINTVL=${LANTERN_CAPQUERY_QRYINTVL}
       - LANTERN_EXPORT_NUMWORKERS=${LANTERN_EXPORT_NUMWORKERS}
       - LANTERN_EXPORT_DURATION=${LANTERN_EXPORT_DURATION}
+      - LANTERN_PRUNING_THRESHOLD=${LANTERN_PRUNING_THRESHOLD}
```

Comment:
I haven't double checked this yet, but I would have thought you would have to include this for the capability_querier below as well for the environment variable to be read in properly? But maybe I'm missing something...